pkgs/misc/vscode-extensions/default.nix

```diff
@@ -718,6 +724,18 @@ let
         };
       };
 
+      ms-toolsai.jupyter = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "jupyter";
+          publisher = "ms-toolsai";
+          version = "2021.5.745244803";
```

Comment:
This is now 2021.6.819764235

Reply:
Crikey, I'd missed that :D I shall sort that out and give it a test.

Reply:
The Python extension does not seem to want me to update to 2021.6.*  If I force the matter then the 2021.6.* extension does not load in vscode non-insiders-edition which suggests that it's not ready for use outside of insiders for now.  On a non-nixos code install where I let vscode manage its extensions itself, it sticks to 2021.5.*  Indeed if I ask my vscode install on my Debian system to "Install other versions" of jupyter, it doesn't offer the 2021.6.* series at all.

Reply:
Aight, let's ship this then.